templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -42,20 +42,44 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>Same as above - can we update this to say "If you'd like to test the features in <code>ark</code> with an example dataset, leave <code>use_example_dataset</code>  as <code>True</code>. If you're using your own data, set as <code>False</code> ."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>